src/ict/model.py

```diff
@@ -86,6 +77,13 @@ def save_clt(self, cwl_path: StrPath, network_access: bool = False) -> Path:
             yaml.dump(self.to_clt(network_access), file)
         return Path(cwl_path)
 
+    def save_cwl(self, cwl_path: StrPath, network_access: bool = False) -> Path:
+        """Save the ICT as CommandLineTool to a file.
+
+        Alias for `save_clt`.
+        """
+        return self.save_clt(cwl_path, network_access)
```

Comment:
This looks like an infinite recursive call.

Reply:
It's an alias. See that one finishes in `clt` and the other in `cwl`.